skademelding/src/main/resources/openapi.yaml

```diff
@@ -265,6 +254,20 @@ components:
         - Periode
         - Ukjent
       example: Tidspunkt
+    Periode:
+      title: Periode
+      type: object
+      properties:
+        fra:
+          type: string
+          format: date
+        til:
+          type: string
+          format: date
+        sykdomPaavist:
+          description: Dersom periode er valgt, må det også gis en dato for når sykdom ble påvist
```

Comment:
Dersom yrkessykdom, bør det også gis en dato for når sykdommen ble påvist

Reply:
Bra! jeg endrer tekst